.circleci/config.yml

```diff
@@ -118,6 +118,28 @@ jobs:
             - target/release/deps
           key: v4-cargo-cache-darwin-{{ arch }}-{{ checksum "Cargo.lock" }}
 
+  test-rust-nightly:
+    docker:
+      - image: circleci/rust:latest
+
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v4-test-cargo-cache-linux-{{ arch }}-{{ checksum "Cargo.lock" }}
```

Comment:
You probably want to append nightly here (if not the cache key will be the same as for the stable one, and will force both to be updated)